lib/roo_on_rails/railties/database.rb

```diff
@@ -1,7 +1,7 @@
 module RooOnRails
   module Railties
     class Database < Rails::Railtie
-      if ActiveRecord::VERSION::MAJOR >= 4
+      if defined?(ActiveRecord) && ActiveRecord::VERSION::MAJOR >= 4
```

Comment:
How does `ActiveSupport.on_load` work with Railties though...? I'm not sure I even want to hazard a guess!